wagtail/admin/views/pages/delete.py

```diff
@@ -24,12 +25,38 @@ def delete(request, page_id):
 
         next_url = get_valid_next_url_from_request(request)
 
+        pages_to_delete = {page}
+
+        # The `construct_synced_page_tree_list` hook returns translation and
+        # alias pages when the action is set to "delete"
+        if getattr(settings, "WAGTAIL_I18N_ENABLED", False):
+            for fn in hooks.get_hooks("construct_synced_page_tree_list"):
+                fn_pages = fn([page], "delete")
+                if fn_pages and isinstance(fn_pages, dict):
+                    for additional_pages in fn_pages.values():
+                        pages_to_delete.update(additional_pages)
+
+        pages_to_delete = list(pages_to_delete)
+        translation_count = len(
+            [p.id for p in pages_to_delete if not p.alias_of and p.id != page.id]
```

Comment:
```suggestion
            [p.id for p in pages_to_delete if not p.alias_of_id and p.id != page.id]
```

Also best to move this next to `"translation_count":` bit of the context in `TemplateResponse`